core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/tracker/UserStoreManagerRegistry.java

```diff
@@ -111,4 +113,39 @@ public static Properties getUserStoreProperties(String className) {
         properties = getUserStoreManagers().get(className);
         return properties;
     }
+
+    /**
+     * Get if the user store is local or not.
+     *
+     * @param className User store name.
+     * @return boolean true if user store is a local.
+     */
+    public static boolean isLocalUserStore(String className) throws UserStoreException {
```

Comment:
Method comment is not complete. We need to add exception information as well. WDYT?

Reply:
fixed https://github.com/wso2/carbon-kernel/pull/3087